app/assets/stylesheets/common/base/search-menu.scss

```diff
@@ -403,18 +423,14 @@ $search-pad-horizontal: 0.5em;
     align-items: center;
 
     .spinner {
-      width: 12px;
-      height: 12px;
-      border-width: 2px;
-      margin: 0 0.5em 0.25em 0;
-      margin-top: 2px;
+      width: 0.75rem; // 12px
```

Comment:
I don't think you need to comment the size like this that will be very verbose if we do this on the whole codebase

Reply:
I will remove it in the following PR. Sorry, I overlooked this comment before merging.